tests/ihontaryk/homework_9/test_frenchdeck.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from ihontaryk.homework_9.frenchdeck import FrenchDeck
+
+
+@pytest.fixture
+def deck():
+    return FrenchDeck()
+
+
+@pytest.mark.parametrize('index, expected_result',
+                         [(1, ('3', 'spades')),
+                          (slice(42, 52, 2), [('5', 'hearts'),
+                                              ('7', 'hearts'),
+                                              ('9', 'hearts'),
+                                              ('J', 'hearts'),
+                                              ('K', 'hearts'),
+                                              ]),
+                          (slice(3), [('2', 'spades'),
+                                      ('3', 'spades'),
+                                      ('4', 'spades')]),
+                          ])
+def test_getitem(deck, index, expected_result):
+    """
+    verify getitem function
+    """
+
+    assert deck[index] == expected_result
+
+    @pytest.mark.parametrize('index, new_value, expected_result',
+                             [(0, ('K', 'hearts'),
+                               ('K', 'hearts')),
+                              (slice(1, 3), [('5', 'hearts'),
+                                             ('7', 'hearts'),
+                                             ('9', 'hearts')],
+                               [('5', 'hearts'),
+                                ('7', 'hearts'),
+                                ('9', 'hearts')
+                                ])
+                              ])
```

Comment:
poor readability of a structure, at least segregete case from case by empty line

Reply:
if new value matches expected result why do you need both